autoagents/eval/README.md

```diff
@@ -0,0 +1,8 @@
+Use test.py to evaluate models:
+
+```
+PYTHONPATH=`pwd` python autoagents/eval/test.py --model gpt-4  --temperature 0 --agent ddg --persist-logs --dataset bamboogle --eval
```

Comment:
Maybe also add the command for hotpotqa evaluation:
```bash
python test.py --model <model_name>  --temperature 0 --agent wiki --persist-logs --dataset hotpotqa --eval
```

Reply:
added